CHANGELOG.md

```diff
@@ -1,87 +1,87 @@
 <a name="v3.3.1"></a>
-# [3.3.1](https://github.com/sjPlot/Zettelkasten/compare/3.3.0...v3.3.1) (2020-04-06)
+# [3.3.1](https://github.com/Zettelkasten-Team/Zettelkasten/compare/3.3.0...v3.3.1) (2020-04-06)
 ### Fehlerbehebungen
-- Look-and-Feel Seaglass wurde entfernt, da es zu Kompatibilitätsproblemen mit JDK 11 führte ([210](https://github.com/sjPlot/Zettelkasten/issues/210))
+- Look-and-Feel Seaglass wurde entfernt, da es zu Kompatibilitätsproblemen mit JDK 11 führte ([210](https://github.com/Zettelkasten-Team/Zettelkasten/issues/210))
```

Comment:
https://github.com/Zettelkasten-Team/Zettelkasten/issues/210 will result in a 404 Page not found
Do we really want to move all the issues to Zettelkasten-Team

Reply:
Ah yes, right, good point. I don't think the issues should be copied or moved. I will correct that.

Reply:
fixed the issue links